product.json

```diff
@@ -47,6 +47,10 @@
 	"gettingStartedUrl": "https://go.microsoft.com/fwlink/?linkid=862039",
 	"releaseNotesUrl": "https://go.microsoft.com/fwlink/?linkid=875578",
 	"documentationUrl": "https://go.microsoft.com/fwlink/?linkid=862277",
+	"updateMetadataUrl": "https://go.microsoft.com/fwlink/?linkid=2274438",
+	"updateMetadataMacUrl": "https://go.microsoft.com/fwlink/?linkid=22745",
```

Comment:
The mac links all seem to be not valid - is that expected?

Reply:
Thanks for catching this! I accidentally deleted two characters when doing the bulk edit. Updated to the correct links and verified the update url via builds here:
https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=226674&view=results